core/pom.xml

```
@@ -20,8 +20,19 @@
         <dependency>
             <groupId>me.aurium</groupId>
             <artifactId>beetle-core</artifactId>
-            <version>${core.version}</version>
+            <version>0.4.1</version>
         </dependency>
     </dependencies>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>3.0.0-M5</version>
```

```suggestion
                <version>3.0.0-M5</version>
                <configuration>
                    <trimStackTrace>false</trimStackTrace>
                </configuration>
```
trimStackTrace should be off by default in the maven-surefire-plugin. It's so annoying.

---

Why is that even an option? Stacktraces should be full, without exception.